src/core/textures/BaseTexture.js

```diff
@@ -405,15 +499,24 @@ BaseTexture.fromImage = function (imageUrl, crossorigin, scaleMode)
             image.crossOrigin = determineCrossOrigin(imageUrl);
         }
 
+        var imageType = getImageTypeOfUrl(imageUrl);
+
+        if (imageType) {
+            image.type = imageType;
+        }
+
         baseTexture = new BaseTexture(image, scaleMode);
         baseTexture.imageUrl = imageUrl;
 
-        image.src = imageUrl;
-
-        utils.BaseTextureCache[imageUrl] = baseTexture;
+        baseTexture.sourceScale = sourceScale;
```

Comment:
That is the scale for the svg image. One can use it to scale the svg before creating the texture. For example in the demo I scale the image based on screen width like: `beeTexture = new PIXI.Texture.fromImage('bee.svg', undefined, undefined, width / 600)`.